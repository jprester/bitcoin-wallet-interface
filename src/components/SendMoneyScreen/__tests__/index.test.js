import React from 'react';
import { shallow } from 'enzyme';
import { SendMoneyScreen } from '../index';

const wrapper = shallow((<SendMoneyScreen startCoinFetch={jest.fn()} />));

describe('index', () => {
  it('should render SendMoneyScreen correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
