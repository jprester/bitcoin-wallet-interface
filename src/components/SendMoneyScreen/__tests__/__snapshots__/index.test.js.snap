// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render SendMoneyScreen correctly 1`] = `
<div
  className="content-block vertical-divider"
>
  <Connect(WithStyles(SendCryptoForm)) />
  <Connect(WithStyles(Component)) />
  <PriceTrackerWidget />
</div>
`;
